common/dds_specific/srcC/DynamicDataFunc.c

```diff
@@ -0,0 +1,85 @@
+/******************************************************************************/
```

Comment:
Minor: I would rename this file and the associated header file to `DynamicDataHelpers.c`. "Func" is a bit obscure.